contracts/amm/AmmFactory.sol

```diff
@@ -113,6 +113,21 @@ contract AmmFactory is OwnableUpgradeable, Proxiable {
         emit AmmImplementationUpdated(ammImplementation);
     }
 
+    /**
+     * The owner can update the AMM implementation address that will be used for future AMMs
+     */
+    function updateAddressesProviderImplementation(
```

Comment:
I think it should be called `updateAddressProviderAddress`. This updates the actual proxy address, not the implementation.

Reply:
ohhhh good call I agree